liberapay/models/community.py

```diff
@@ -91,3 +101,14 @@ def check_membership_status(self, participant):
     @property
     def nsubscribers(self):
         return self.participant.nsubscribers
+
+    @staticmethod
+    def _unconfusable(name):
+        unconfusable_name = ''
+        for c in name:
+            confusable = confusables.is_confusable(c, preferred_aliases=['COMMON', 'LATIN'])
+            if confusable:
+                # if the character is confusable we replace it with the first prefered alias
+                c = confusable[0]['homoglyphs'][0]['c']
+            unconfusable_name += c
+        return unconfusable_name
```

Comment:
This function isn't specific to communities, so it should probably be in the `liberapay.utils` module instead of in the `Community` class.

Reply:
Fixed here: 9b496bc88f8a3c404394ee8c1999e33a6f944a0b